package.json

```diff
@@ -107,7 +107,6 @@
     "eslint-plugin-node": "^9.0.1",
     "fastboot": "2.0.1",
     "foundation-sites": "6.5.0-rc.4",
-    "husky": "^6.0.0",
```

Comment:
I removed this as a temporary convenience but happy to re-add it. As far as I could tell it was not working. If it's valuable to the team still then we can add it but it seems redundant to the current CI steps.

Reply:
It is redundant with CI but it's intentional redundancy 😄 that being said, fine with removing it until we get things upgraded and then revisit it. I think you're correct that it's not quite set up correctly anyway.